doc/source/getting_started/WorkingWithInferenceData.ipynb

```diff
@@ -13,11 +13,13 @@
   },
```

Comment:
<p>The explanation should not be as a comment but in a cell above, also with a title so it shows up in the right toc, same for all code samples.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz/pull/1697/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>